cfgov/v1/models/snippets.py

```diff
@@ -9,19 +9,31 @@
 
 @register_snippet
 class Contact(models.Model):
-    heading = models.CharField(max_length=22, blank=False)
+    title = models.CharField(verbose_name=('Title'), max_length=255,
+                             help_text=("The snippet title as you'd like it to be seen by the public"))
+    slug = models.SlugField(verbose_name=('Slug'), max_length=255,
```

Comment:
@kurtw A BaseSnippet class isn't necessary, since a snippet can hold any kind of data it was just coincidence that WP contacts share some fields similar to pages like 'title' and 'slug' 

but it doesn't mean future snippets will hold the same information too